source/features/conflict-marker.tsx

```diff
@@ -1,63 +1,42 @@
 import './conflict-marker.css';
 import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
 import select from 'select-dom';
 import {AlertIcon} from '@primer/octicons-react';
-import oneMutation from 'one-mutation';
-import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import api from '../github-helpers/api.js';
+import observe from '../helpers/selector-observer.js';
 
-type PRConfig = {
-	number: string;
-	user: string;
-	repo: string;
-	link: HTMLAnchorElement;
-	key: string;
-};
-
-function createQueryFragment(pr: PRConfig): string {
+// NOTE: cannot separate to gql: https://github.com/refined-github/refined-github/pull/6839#issuecomment-1683684796
+function createQueryFragment(prKey: string, owner: string, name: string, prNumber: number): string {
 	return `
-		${pr.key}: repository(owner: "${pr.user}", name: "${pr.repo}") {
-			pullRequest(number: ${pr.number}) {
+		${prKey}: repository(owner: "${owner}", name: "${name}") {
+			pullRequest(number: ${prNumber}) {
 				mergeable
 			}
 		}
 	`;
 }
 
-function buildQuery(prs: PRConfig[]): string {
-	return prs.map(pr => createQueryFragment(pr)).join('\n');
-}
+async function addConflictMarkers(container: HTMLDivElement): Promise<void> {
+	const links = select.all('.js-issue-row:has(.octicon-git-pull-request.color-fg-open) a.js-navigation-open', container);
```

Comment:
The feature will likely fail when there are 0 links

Reply:
Thank you! It was fixed.